### PR TITLE
Deps updates

### PR DESCRIPTION
**Pull request by spirali**

### CODE REVIEW

**Comment by spirali** [comment]
hasbrown & dev dependencies not updated (I do not have energy to fix it:)

File: crates/hyperqueue/src/worker/parser.rs [hunk 1]
```diff
@@ -564,7 +564,7 @@ mod test {
     #[test]
     fn test_parse_resource_def_numeric_value() {
         insta::assert_snapshot!(expect_parser_error(parse_resource_definition_inner(), "x=1"), @r###"
-        Unexpected end of input found while attempting to parse resource kind, expected _ or x:
```
Comment (Kobzol) [inline crates/hyperqueue/src/worker/parser.rs:567]: This is an error regression :(

**Review by Kobzol — APPROVED** [review]

File: crates/hyperqueue/src/worker/parser.rs (hunk 1 above)
Comment (spirali) [reply crates/hyperqueue/src/worker/parser.rs:567]: Do you want to look at? Otherwise I will just revert to the older version.

Comment (Kobzol) [reply crates/hyperqueue/src/worker/parser.rs:567]: Nah, let's just leave it as it is, the error messages are quite imprecise in other locations too, anyway.

Comment (spirali) [reply crates/hyperqueue/src/worker/parser.rs:567]: Leave it in the updated version version?

Comment (Kobzol) [reply crates/hyperqueue/src/worker/parser.rs:567]: Yeah let's keep 0.9, it should be faster to compile and includes some nice improvements.

**Pull request merged**